components/aws/athena/query/component.yaml

```
@@ -0,0 +1,35 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Athena - Query
+  A Kubeflow Pipeline component to submit a query to Amazon Web Services Athena
+  service and dump outputs to AWS S3.
+
+inputs:
+  - {name: region,        description: 'The Athena region in which to handle the request.'}
+  - {name: database,      description: 'The name of the database.'}
+  - {name: query,         description: 'The SQL query statements to be executed in Athena.}
```

The tests a failing - missing ending quote.
```suggestion
  - {name: query,         description: 'The SQL query statements to be executed in Athena.'}
```

---

Thanks! I miss this quota in latest commit.